suites/pacific/rados/tier_3_rados_scrub.yaml

```diff
@@ -70,3 +70,32 @@ tests:
       desc: Scheduled scrub validation
       config:
         scenario: "beginTime gt endTime lt currentTime"
+  - test:
+      name: Begin Time & End time > current
+      polarion-id: CEPH-9367
+      module: scheduled_scrub_scenarios.py
+      desc: Scheduled scrub validation
+      config:
+        scenario: "beginTime and endTime gt currentTime"
+
+  - test:
+      name: Begin Time & End time > current
+      polarion-id: CEPH-9368
+      module: scheduled_scrub_scenarios.py
+      desc: Scheduled scrub validation
+      config:
+        scenario: "beginTime and endTime gt currentTime"
```

Comment:
This and the above scenario ( ` beginTime and endTime gt currentTime `) appear to be same with different polarion ID's. Is that intended?

Reply:
Looking into it.

Reply:
The scenario which it missed is  B & E < C where 
B= Begin Time
E = End Time
C= Current Time
This scenario I am covering in the test case where B>E and C< E. In this, both B & E are less than C but B >E.
I will modify it according to this.